RocketChat/Rocket.Chat
# [IMPROVE] Rewrite Save Encryption Password Modal

## Proposed changes (including videos or screenshots)


### before
![image](https://user-images.githubusercontent.com/27704687/122980201-c337a800-d36e-11eb-8e2b-68534cea8e1e.png)

### after
![image](https://user-images.githubusercontent.com/27704687/122980409-f8dc9100-d36e-11eb-9c15-aff779c84a91.png)



## Issue(s)


## Steps to test or reproduce

- Go to MyAccount 
- Go to the security menu
- Reset the E2E Key on the E2E Encryption tab
- After reloading the page you will able to see the banner "Save Your Encryption Password"
- Click on it to open the new Modal

## Further comments
